js-runner/README.md

```diff
@@ -0,0 +1,51 @@
+# @grain/js-runner
+
+The JavaScript runner for the Grain language.
+
+## Why
+
+Grain programs need to be instantiated in a specific way within JavaScript environments. By using `@grain/js-runner`, you can bootstrap the environment to run your Grain programs.
```

Comment:
This is true for unlinked mode but otherwise it's largely just setting up WASI, which can be done without the runner. I'd probably say something more along the lines of this makes it easier to get Grain programs running in JS rather than that Grain programs are special.

Reply:
Can someone please make an edit suggestion. My brain is fried today.